src/webots/scene_tree/WbAddNodeDialog.cpp

```diff
@@ -673,6 +685,30 @@ void WbAddNodeDialog::import() {
   accept();
 }
 
+bool WbAddNodeDialog::isAmbiguousProto(const QString &protoName, const QString &url) {
+  if (!mUniqueLocalProto.contains(protoName))
+    return false;
+  if (mUniqueLocalProto.value(protoName) == url)
+    return false;
+  // the url might differ, but they might point to the same object (ex: one is relative, the other absolute)
+  QString thisUrl;
+  if (WbUrl::isLocalUrl(thisUrl))
+    thisUrl = QDir::cleanPath(WbStandardPaths::webotsHomePath() + url.mid(9));
+  else
+    thisUrl = mUniqueLocalProto.value(protoName);
+
+  QString otherUrl;
+  if (WbUrl::isLocalUrl(thisUrl))
+    thisUrl = QDir::cleanPath(WbStandardPaths::webotsHomePath() + url.mid(9));
+  else
+    thisUrl = url;
```

Comment:
Seems wrong (didn't check clang-format):
```suggestion
  const QString thisUrl = WbUrl::isLocalUrl(url) ? QDir::cleanPath(WbStandardPaths::webotsHomePath() + url.mid(9)) : mUniqueLocalProto.value(protoName);

  const QString otherUrl = WbUrl::isLocalUrl(url) ? QDir::cleanPath(WbStandardPaths::webotsHomePath() + url.mid(9)) : url;
```